docs/installation.md

```
@@ -81,21 +81,21 @@ sudo make install # Installs mcpd 'globally' to /usr/local/bin
 
 ### Default environment variables
 
-| Name                | Default Value                              |
-|---------------------|--------------------------------------------| 
-| `MCPD_API_PORT`     | `8090`                                     |
-| `MCPD_LOG_LEVEL`    | `info`                                     |
-| `MCPD_LOG_PATH`     | `/var/log/mcpd/mcpd.log`                   |
-| `MCPD_CONFIG_FILE`  | `/etc/mcpd/.mcpd.toml`                     |
-| `MCPD_RUNTIME_FILE` | `/home/mcpd/.config/mcpd/secrets.prd.toml` |
+| Name                | Default Value                               |
+|---------------------|---------------------------------------------| 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)